src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -292,10 +293,20 @@ public IActionResult GetAdminToken()
         [HttpGet]
         [HttpPost]
         [Route("admin/warmup")]
-        [Authorize(Policy = PolicyNames.AdminAuthLevelOrInternal)]
         [RequiresRunningHost]
         public async Task<IActionResult> Warmup([FromServices] WebJobsScriptHostService hostService)
         {
+            // Endpoint only for Windows Elastic Premium or Linux App Service plans
+            if (!(_environment.IsLinuxAppService() || _environment.IsWindowsElasticPremium()))
+            {
+                return NotFound();
+            }
+
+            if (Interlocked.CompareExchange(ref _warmupExecuted, 1, 0) != 0)
+            {
+                return Ok();
```

Comment:
Don't you have to update the tests? Probably should be failing currently with these changes due to incorrect sku?

Reply:
Yeah test did fail. Will update them. Made the PR quickly to get feedback and discuss next steps in the meanwhile.